arcsi/api/item.py

```diff
@@ -457,6 +298,7 @@ def listen_play_file(id):
 
 
 @arcsi.route("/item/<id>/download", methods=["GET"])
+@roles_required("admin", "host", "guest")
```

Comment:
@tuz666 why is that? We could use `@auth_token_required` here too, no?

Reply:
This one is clear. `@roles_required` only means that we don't assume if it's logged in user or an API call. Note: on site registrations will not assign a role (not even `guest`) so this one is a real restriction (do `admin`, `host` or `guest`).

Reply:
This was also a good question, I thought this means that if you have any of the listed roles than have the authentication, but it turned out that you should have each of them.